companion/src/firmwares/edgetx/yaml_customfunctiondata.cpp

```diff
@@ -65,6 +65,7 @@ static const YamlLookupTable customFnLut = {
   {  FuncScreenshot, "SCREENSHOT"  },
   {  FuncRacingMode, "RACING_MODE"  },
   {  FuncDisableTouch, "DISABLE_TOUCH"  },
+  {  FuncDisableAudioAmp, "DISABLE_AUDIO_AMP"  },
```

Comment:
For consistency with enum move to end of list

Reply:
done